spec/models/lockstep/api_record/crud_spec.rb

```diff
@@ -99,19 +99,18 @@
   describe "#bulk_import" do
     it "should return false and have error messages attached if the request is invalid" do
       VCR.use_cassette("models/lockstep/contacts/bulk_import_invalid") do
-        obj1 = Lockstep::Contact.new email_address: "a@b.com" # Valid object
-        obj2 = Lockstep::Contact.new email_address: "a" # Invalid object
-        expect(Lockstep::Contact.bulk_import([obj1, obj2])).to be_falsey
-        expect(obj1.errors.full_messages.blank?).to be_truthy
-        expect(obj2.errors.full_messages.present?).to be_truthy
+        Lockstep::Client.set_api_key('LSPK-PC/Ywx1A0tJz4QKmbuu5SRj5W0PLxRy4ZDQZriNM/kIG93Ci2u7MBjomzIktix4joPn4zkERt2ctazNULLWopw==')
```

Comment:
We shouldn't be setting API key explicitly as this is a public repo. We should re-use the auth context which handles this.

Reply:
done